LICENSE

```diff
@@ -222,6 +222,7 @@ BSD 2-clause License
 
 3rdparty/picojson
 3rdparty/dmlc-core/include/dmlc/concurrentqueue.h
+python/tvm/_ffi/decorator.py
```

Comment:
sorry i missed this part. Can we simply remove decorator dep via functools or do we still need the original decorator impl here? Would be good to understand, ideally we either remove, or keep the thirdparty dep for a bit

Reply:
would be good to get a testcases that altnerative approach failed to understand here